users/users/user.php

```diff
@@ -0,0 +1,46 @@
+<?php
```

Comment:
@coolbung should resource name be search?

Reply:
I the search API is nothing but the list view with the additional filters. Can we merge this code with users post API? cc @coolbung

Reply:
Yes. Please go ahead as discussed. 

Once merged, moved out the plugins in the respective repos. Any plugin that does not have a specific repo can stay here.